develop.md

```diff
@@ -8,7 +8,7 @@ Releasing a new version of an image
 
 Create the new image for the new version:
 
-1. Rename the last version's directory, for example `7.7-community` to `7.8-community`
+1. Rename the last version's directory, for example `7.8-community` to `7.9-community`
```

Comment:
7.9 doesn't exists yet

Reply:
Yes this is the documentation to create an image for the **next** version which would be 7.9.